### PR TITLE
[13.0][IMP] sale_partner_selectable_option: Improve sale button in contact form to show only if "Selectable in orders" is checked.

### PR DESCRIPTION
**Pull request by victoralmau**
Improve sale button in contact form to show only if "Selectable in orders" is checked.

Please @ernestotejeda and @pedrobaeza can you review it?

@Tecnativa TT33933

### CODE REVIEW

**Review by ernestotejeda — APPROVED** [review]

File: sale_partner_selectable_option/views/res_partner_view.xml [hunk 1]
```diff
@@ -11,4 +11,16 @@
             </group>
         </field>
     </record>
+    <record id="res_partner_view_buttons" model="ir.ui.view">
+        <field name="name">res.partner.view.buttons</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="sale.res_partner_view_buttons" />
+        <field name="arch" type="xml">
+            <xpath expr="//button" position="attributes">
```
Comment (pedrobaeza) [inline sale_partner_selectable_option/views/res_partner_view.xml]: Please be more specific in the button to hide.

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Can you try again? (sometimes some test fails).

**Pull request merged**